comp/netflow/server/listener.go

```diff
@@ -17,18 +18,31 @@ type netflowListener struct {
 	flowState *goflowlib.FlowStateWrapper
 	config    config.ListenerConfig
 	error     *atomic.String
+	flowCount *atomic.Int64
+}
+
+func (l *netflowListener) listen(inputChan <-chan *common.Flow, flowAgg *flowaggregator.FlowAggregator) {
+	for flow := range inputChan {
+		l.flowCount.Add(1)
+		flowAgg.GetFlowInChan() <- flow
+	}
 }
 
 func startFlowListener(listenerConfig config.ListenerConfig, flowAgg *flowaggregator.FlowAggregator, logger log.Component) (*netflowListener, error) {
 	atomicErr := atomic.NewString("")
+	flowCount := atomic.NewInt64(0)
+	inputChan := make(chan *common.Flow)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Shouldn't this channel be buffered?